azext_iot/central/providers/device_provider.py

```diff
@@ -190,6 +202,108 @@ def delete_device(
 
         return result
 
+    def list_relationships(
+        self,
+        device_id,
+        rel_name=None,
+        central_dns_suffix=CENTRAL_ENDPOINT,
+    ) -> List[RelationshipV1_1_preview]:
+        relationships = central_services.device.list_relationships(
+            self._cmd,
+            app_id=self._app_id,
+            device_id=device_id,
+            token=self._token,
+            api_version=self._api_version,
+            central_dns_suffix=central_dns_suffix,
+        )
+
+        if relationships is None:
+            raise ResourceNotFoundError(
```

Comment:
Since this is a list operation, it makes more sense to just return [] to show that there are no relationships

Reply:
+1